app/packages/core/src/components/applications/Applications.tsx

```diff
@@ -0,0 +1,207 @@
+                      ))}
+                    </Box>
+                  )}
+                  {application.teams && application.teams.length > 0 && (
+                    <Chip
+                      size="small"
+                      icon={<PeopleOutlined />}
+                      sx={{ cursor: 'pointer' }}
+                      label={application.teams.length === 1 ? '1 Team' : `${application.teams.length} Teams`}
+                    />
+                  )}
+                  {application.topology &&
+                    application.topology.dependencies &&
+                    application.topology.dependencies.length > 0 && (
+                      <Chip
+                        size="small"
+                        icon={<HubOutlined />}
+                        sx={{ cursor: 'pointer' }}
+                        label={
+                          application.topology.dependencies.length === 1
+                            ? '1 Dependency'
+                            : `${application.topology.dependencies.length} Dependencies`
+                        }
+                      />
+                    )}
+                  {application.topology && application.topology.external && application.topology.external === true && (
+                    <Chip size="small" icon={<HubOutlined />} sx={{ cursor: 'pointer' }} label="External Application" />
+                  )}
+                </Box>
+              ) : null}
+            </>
+          }
+        />
+      </ListItem>
+
+      <ApplicationInsights application={application} onClose={hideInsights} open={open} />
+    </>
+  );
+};
+
+interface IApplicationsProps {
+  options: IApplicationOptions;
+  setOptions: (options: IApplicationOptions) => void;
+}
+
+/**
+ * The `Applications` component load the applications for the provided `options` and renders them as a list where each
+ * list item is seperated by a `Divider` component. Each list item is rendered via the `Applications` component. If the
+ * number of possible applications for the filter options is larger then the returned applications a pagination
+ * component will be shown, which can be used to get the other applications which are not shown.
+ *
+ * The returned applications are cached with the provided options as key, so that we do not have to reload the
+ * applications when a user wants to view the applications for the same filter twice or more.
+ */
+const Applications: FunctionComponent<IApplicationsProps> = ({ options, setOptions }) => {
+  const apiContext = useContext<IAPIContext>(APIContext);
+
+  const { isError, isLoading, error, data, refetch } = useQuery<
+    { applications?: IApplication[]; count?: number },
+    APIError
+  >(['core/applications/applications', options], async () => {
+    const c = options.clusters?.map((cluster) => `&cluster=${encodeURIComponent(cluster)}`);
+    const n = options.namespaces?.map((namespace) => `&namespace=${encodeURIComponent(namespace)}`);
```

Comment:
params are not correct (either change them here or in `pkg/hub/api/applications/applications.go`

```suggestion
    const c = options.clusters?.map((cluster) => `&clusterID=${encodeURIComponent(cluster)}`);
    const n = options.namespaces?.map((namespace) => `&namespaceID=${encodeURIComponent(namespace)}`);
```